gh-102812: unittest's addError/addFailure/addSubTest accept an exception instance in addition to (typ, val, tb) as paramter (#102814)
python/cpython: Misc/NEWS.d/next/Library/2023-03-18-13-09-35.gh-issue-102812.IcPYBs.rst

:meth:`~unittest.TestResult.addError`,
:meth:`~unittest.TestResult.addFailure` and

> :meth:`~unittest.TestResult.addSubTest` now accept either an exception
> instance or a ``(typ, val, tb)`` tuple.

Reviewer: ```suggestion
:meth:`~unittest.TestResult.addSubTest` now accept an exception instance
instead of a ``(typ, val, tb)`` tuple.
```
